src/signals.js

```diff
@@ -8,6 +8,6 @@ export const CLOSE = prefix + 'CLOSE';
 export const NAVIGATE = prefix + 'NAVIGATE';
 
 // Creators
-export const open = createAction(OPEN);
+export const open = createAction(OPEN, (route, params) => ({route, params}));
```

Comment:
@jackmellis this is a API breaking change. To keep it from breaking over projects for no reason then keep the signature the same. Aka, take the route and params in as an object.